src/main/java/chess/domain/game/GameCommand.java

```diff
@@ -43,14 +47,24 @@ private void validateMoveCommand(String source, String target) {
         }
     }
 
-    public boolean isStart() {
-        return gameCommand.get(GameCommandIndex.COMMAND.index).equals(START_COMMAND);
+    public void isStart() {
+        if (!gameCommand.get(GameCommandIndex.COMMAND.index).equals(START_COMMAND)) {
```

Comment:
```suggestion
        if (!START_COMMAND.equals(gameCommand.get(GameCommandIndex.COMMAND.index))) {
```
위와 같이 작성하면 NPE를 방어할 수 있습니다.

Reply:
이 부분도 슬랙을 통해 얘기 나눈 것을 통해 전체적인 equals문을 점검하여 문자열을 먼저하도록 리팩토링 하였습니다 ˙ᵕ˙